### DISCUSSION TITLE
Backing up Mongo under Debian, Changelog, and Clipboard Issues

### DISCUSSION THREAD

**Discussion by samschultzponsys** [Ylianst/MeshCentral#5777] [Issues] [answered]
I am seeing there is a newer version of Mesh available but I do not see any changelogs listed anywhere and I would love to keep tabs on whats been updated. I am using mongodb with my instance of mesh but I do not know what the path or command under debian would be in my config.json to have a database backup dumped would be, any help or example here for pathing would be great as all the gudes for mesh im seeing use windows file structure. Lastly I am noticing a lot of really flaky clipboard behavior and I am not sure if this is just normal or if I should be making a issue request to have it looked at. Thank you for any assistance.

**Comment by si458** [comment, marked as answer]
change logs are all here (but it really just every commit we did) - https://github.com/Ylianst/MeshCentral/releases
EDIT: you can use a service like https://newreleases.io/github/ylianst/meshcentral to get email alerts on new releases for things
![image](https://github.com/Ylianst/MeshCentral/assets/765314/e76bc7c7-62ba-46cc-82c0-16aa7e707607)
![image](https://github.com/Ylianst/MeshCentral/assets/765314/20705423-b92f-4713-a21a-e338dec4a38a)

**Reply by samschultzponsys** [reply]
Thank you! This is what I was looking for.
